Our-Project.ipynb

```diff
@@ -7,6 +7,111 @@
     "# This is the start!!!"
```

Comment:
<p>Great job on identifying the common genres. I think this is a great start. As the above 2 cell blocks does somewhat the same functions, shall we just add the <code>common_genres</code>  function to the first cell block? Basically merge the changes you made to the <code>processed_list</code>  to the first cell block. </p><p> </p><p>One other suggestion I'd like to add is that for pre-processing, i think it would be <strong>clean</strong> if we could export the relevant rows that we are not using from <code>MovieGenre.csv</code>  to a separate csv file. This is so we can show that we cleanly separated the irrelevant data in the pre-processing stage. Lmk what you think haha   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CS3244-ML-AY2021S1-0037/CS3244-ML/pull/11/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>1. There is some parallel operation in the 2nd block, so I wrote the first block to filter the top 10 genres.</p><p>2.Yes, I already did that, its being exported to the ProcessedData.csv</p>

Reply:
1. Ah okay. I did not see that you were cross checking the `final_genres` array you created in the first cell with the list of genres in the second cell. I guess it had to be done to verify the data. Out of curiosity, were there any instances that were found in? 
```
if common == []:
            return
``` 
One last thing i thought of mentioning was to have a counter in the first cell to check if 39998 images were checked through. Its just to sort of double verify it lol. Sorry if im being anal lol. Lmk if you think otherwise.

2. Okay actually what i meant was to create a `missingImages.csv` but it seems like its already been done. Was that from the dataset or did we create that ourselves?

Reply:
final_genre is the top 10 genres, genre is the labels, if they have nth in common => row is filtered out

Reply:
yeah i know that -- i was just wondering if there were any rows filtered out?

Reply:
Yup

Reply:
oh my god i just got it lol

Reply:
my bad

Reply:
yeah this is ready to be merged in. Let me know if you have any idea where `missingImages.csv` came from

Reply:
That was not created by me, can ask in the group